org-super-agenda.el

```diff
@@ -647,6 +647,30 @@ available."
           (_ (cl-loop for fn in args
                       thereis (funcall fn item)))))
 
+(org-super-agenda--defgroup property
+  "Group items that contain a property with a given value.
+Argument should be a list with a string representing a
+property, and an optional second argument being either a string
+representing a value, or a function which will be passed the
+found property value, if any. Without a second argument, any item
+with the given property will match. "
+  :section-name (concat "Property: '" (car args) "'"
```

Comment:
The extra `'` characters look cluttered to me.  I'd suggest using this format: `Property: property: value`.

Reply:
Here's some code that sets the section name:
```el
(concat "Property: " (car args)
        (pcase (cadr args)
          (`nil nil)
          ((pred stringp) (concat ": " (cadr args)))
          ((and (pred functionp) (pred symbolp))
           (concat " matches predicate " (symbol-name (cadr args))))
          ((pred functionp) (concat " matches lambda predicate"))))
```

Reply:
Again, I have now shamelessly copied your suggestion :)